src/components/common/markets/dashboard-group.tsx

```
@@ -80,90 +79,99 @@ export const DashboardGroup = ({ markets, type, categoryName }: DashboardGroupPr
         )
 
       case DashboardGroupType.Featured:
-        const featuredMarkets = markets.slice(0, 9)
         const rowStructure = [3, 2, 3, 1]
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)